sql/src/main/java/io/crate/metadata/rule/ingest/IngestRule.java

```diff
@@ -54,6 +54,10 @@ public String getTargetTable() {
         return targetTable;
     }
 
+    public void setTargetTable(String targetTable) {
```

Comment:
well, we can (and I'm all for immutability), but is it really worth it in this case? The target table is not part of the IngestRule identity (it's the name the identifies the rule). It feels to me like we'd be creating loads of garbage